content/docs/contributing/to-docs/get-started.md

```diff
@@ -1,108 +1,231 @@
+
+1. Verify that the changed files contains only the changes you want to submit. Only the commits you made should be listed. If there are more commits than you expected, stop here and make sure to properly sync your branch.
+
+1. Choose the green **Create pull request** button to open the **Open a pull request** page. 
+
+1. Add a descriptive title and description. Be sure to include the issue number in the title of the PR if the PR is addressing an existing issue.
+
+1. In **Reviewers**, add **o3de/docs-reviewers** so they can review your PR. Add additional reviewers as appropriate. If you are submitting technical docs, add reviewers who can verify the technical accuracy of your docs. 
+
+1. Choose the **Create pull request** to submit the PR.
+
+If you go to the main o3de.org repo on GitHub and refer to [O3DE repository pull requests (PRs)](https://github.com/o3de/o3de.org/pulls), your new PR appears at the top of the list.
+
+For more information on creating a pull request, refer to [Creating a pull request](https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/creating-a-pull-request).
+
+### Respond to feedback
+
+Feedback comes in the form of comments that you can address by editing files in your local branch, and *suggestions* that can be committed from the GitHub PR interface. It's important to understand that the PR process is a collaborative discussion. Every comment does not need to be addressed, and every suggestion does not need to be integrated. When the required number of reviewers approve of your contribution, it can be integrated. Here are a few tips for addressing suggestions and comments:
+
+* To commit multiple suggestions, use the batch functionality to integrate multiple suggestions in a single commit.
+* Do try to acknowledge all the feedback you are given. This doesn't mean to integrate every comment and commit every suggestion. It simply means to avoid resolving comments and suggestions without action or a response. Embrace collaboration.
+* When addressing comments, maintain the relative conversation in the PR, edit the topics as necessary, and commit the changes to your fork/branch. Your new commits will be automatically added to the PR.
+* Make sure to request a re-review of your new commits if required. More guidance on how to traverse our PR process is coming!
+
+For more information on incorporating PR feedback, refer to [Incorporating feedback in your pull request](https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/incorporating-feedback-in-your-pull-request).
+
+### The PR merge
+
+When you have approval from two reviewers, and approval from a technical reviewer (if necessary), your PR can be merged. Never merge your own PRs. The last reviewer to approve the PR is responsible for merging the PR.
```

Comment:
I believe only Docs maintainers should merge PRs.

Reply:
Yes was understood initially, but not always practiced. Especially since the `write` access was changed to include reviewers from other domains. I'll update this text.